.github/workflows/audit.yml

```diff
@@ -0,0 +1,13 @@
+name: Audit
+
+on: [push, pull_request]
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Install
+        run: npm install --legacy-peer-deps
+      - name: Audit signatures
+        run: npm audit signatures
```

Comment:
What is this?

Reply:
This is an audit action that we have in some repositories that reports package vulnerabilities.